addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials_pbr_metallic_roughness.py

```diff
@@ -58,8 +58,6 @@ def __gather_base_color_factor(blender_material, export_settings):
         base_color_socket = gltf2_blender_get.get_socket(blender_material, "BaseColor")
     if base_color_socket is None:
         base_color_socket = gltf2_blender_get.get_socket_old(blender_material, "BaseColorFactor")
-    if base_color_socket is None:
-        base_color_socket = gltf2_blender_get.get_socket(blender_material, "Background")
```

Comment:
@donmccurdy Are you OK if we drop backwards compatibility to "Background" unlit nodes at the same time as we embrace the new system?

Reply:
Background node support is retained, it's just moved off the main path to the unlit path. In master, if `gltf2_blender_get.get_socket(blender_material, "Background")` returns non-None then the material is unlit.

https://github.com/KhronosGroup/glTF-Blender-IO/blob/2423afdec5f3aeed7637638c878b4736f8773b77/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py#L157-L158

In this PR, detect_shadeless_material does the same check

https://github.com/KhronosGroup/glTF-Blender-IO/blob/4e67a996bcc3f816854731c920b27e45418a088f/addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials_unlit.py#L28-L30

The reason those lines are deleted is they will never run now; if there had been a Background node, it would have gone down the unlit path instead.

I tested this setup and it exports as unlit with this branch, same as master

![unli](https://user-images.githubusercontent.com/11024420/93538864-ccde3c00-f914-11ea-9c1a-91569cd1ea9b.png)